src/lib/lumi.ts

```diff
@@ -5247,6 +5254,26 @@ export const toZigbee = {
             }
         },
     } satisfies Tz.Converter,
+    lumi_sensitivity_adjustment: {
```

Comment:
Could you use modernExtend for this just like: https://github.com/Koenkk/zigbee-herdsman-converters/blob/9c3b9876056c84a94ae7083a36bedc7b6aafe9ec/src/lib/lumi.ts#L1534